packages/pigeon/platform_tests/ios_swift_unit_tests/ios/RunnerTests/AsyncHandlersTest.swift

```diff
@@ -43,9 +43,12 @@ class AsyncHandlersTest: XCTestCase {
     
     let expectation = XCTestExpectation(description: "voidvoid callback")
     binaryMessenger.handlers[channelName]?(nil) { data in
-      let outputMap = binaryMessenger.codec.decode(data) as? [String: Any]
-      XCTAssertNil(outputMap?["result"])
-      XCTAssertNil(outputMap?["error"])
+      let outputList = binaryMessenger.codec.decode(data) as? [Any]
+      XCTAssertNotNil(outputList)
+      XCTAssertNil(outputList?.first)
+        if (outputList != nil) {
+        XCTAssertFalse(outputList!.count > 1)
```

Comment:
nit: I'd stick with using `XCTAssertTrue` unless you have to use `XCTAssertFalse` since it requires less mental gymnastics.